akka-serialization-jackson/src/main/scala/akka/serialization/jackson/internal/AkkaJacksonSerializationFeature.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2009-2023 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.serialization.jackson.internal
+
+import akka.annotation.InternalApi
+import akka.serialization.jackson.CborSerializable
+import akka.serialization.jackson.JsonSerializable
+import org.graalvm.nativeimage.hosted.Feature
+import org.graalvm.nativeimage.hosted.RuntimeReflection
+
+/**
+ * INTERNAL API
+ */
+@InternalApi
+final class AkkaJacksonSerializationFeature extends Feature {
```

Comment:
👍🏼 great having this automatically handled

Reply:
It could possibly work with a feature for user defined extensions as well, because if a user defines something as an extension it is very likely that they also interact with it in their app, rather than just register it for start through config and then never call it. 

I thought that it is better to document than have something that mostly works but needs extra caveats in case they only register it through `akka.extensions` or `akka.library-extensions`.